### PR TITLE
Add WdlPermission to wdl_options.proto

### PR DESCRIPTION
**Pull request by jbingham-google**
This will allow setting a get and update permission on a trait and a permission on a command.

### CODE REVIEW

File: wdl/wdl_options.proto [hunk 1]
```diff
@@ -514,6 +516,9 @@ message WdlTraitOptions {
   optional uint32 reserved_command_tag_max = 11;
   optional uint32 reserved_event_tag_min = 12;
   optional uint32 reserved_event_tag_max = 13;
+
+  optional WdlPermission get_permission = 20;
+  optional WdlPermission update_permission = 20;
```
Comment (robszewczyk) [inline wdl/wdl_options.proto:521]: Should be different field numbers

File: wdl/wdl_options.proto [hunk 2]
```diff
@@ -586,6 +591,10 @@ message WdlServiceOptions {
   optional WdlResourceOptions resource = 3;
 }
 
+message WdlPermission {
+  optional string permission = 1;
```
Comment (robszewczyk) [inline wdl/wdl_options.proto:595]: Is the string the best representation for a permission?

File: wdl/wdl_options.proto (hunk 1 above)
Comment (robszewczyk) [reply wdl/wdl_options.proto:521]: Also, these fields seem more like commands rather than state variables.

**Review by robszewczyk — CHANGES_REQUESTED** [review]

File: wdl/wdl_options.proto (hunk 2 above)
Comment (robszewczyk) [reply wdl/wdl_options.proto:595]: Confirmed it is what was specified.

File: wdl/wdl_options.proto (hunk 1 above)
Comment (jbingham-google) [reply wdl/wdl_options.proto]: Fixed the field tag.

**Review by robszewczyk — APPROVED** [review]

**Pull request merged**